bindata/network/multus/multus.yaml

```diff
@@ -75,12 +77,17 @@ data:
       sourcedir=$DEFAULT_SOURCE_DIRECTORY
     fi
 
-    cp -rf ${sourcedir}* $DESTINATION_DIRECTORY
+    if [ ! -z "$IGNORE_FILES" ]; then
+      cp -rf ${sourcedir}!($IGNORE_FILES) $DESTINATION_DIRECTORY
```

Comment:
I don't think we especially need to worry about *not* copying some files (as long as we're copying the right versions of them).

Reply:
Yeah, I did that to keep consistency between cluster "with multus" vs. "without", as mentioned a couple of comments above. But I'll remove it